nemo_text_processing/inverse_text_normalization/inverse_normalize.py

```diff
@@ -102,7 +107,10 @@ def inverse_normalize(self, text: str, verbose: bool) -> str:
 
 def parse_args():
     parser = ArgumentParser()
-    parser.add_argument("input_string", help="input string", type=str)
+    input = parser.add_mutually_exclusive_group()
+    input.add_argument("--text", dest="input_string", help="input string", type=str)
+    input.add_argument("--file", dest="input_file", help="input file path", type=str)
+    parser.add_argument('--out', dest="output_file", help="output file path", type=str)
```

Comment:
could you update the top on the inverse_normalize.py and normalize.py scripts to outline that either `text` or `file` must be provided?

Reply:
we don't have to, argparse will do